src/main/java/oshi/hardware/platform/mac/MacNetworks.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2016 com.github.dblock.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ * enrico[dot]bianchi[at]gmail[dot]com
+ *    com.github.dblock - initial API and implementation and/or initial documentation
+ */
+package oshi.hardware.platform.mac;
+
+import java.net.NetworkInterface;
+import java.net.SocketException;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Enumeration;
+import java.util.List;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import oshi.hardware.common.AbstractNetworks;
+import oshi.hardware.stores.HWNetworkStore;
+
+/**
+ *
+ * @author enrico[dot]bianchi[at]gmail[dot]com
+ */
+public class MacNetworks extends AbstractNetworks {
+
+    private static final Logger LOG = LoggerFactory.getLogger(MacNetworks.class);
+
+    private void setNetworkStats(HWNetworkStore netstore) {
+        // TODO: set network stats on Mac
```

Comment:
For now, this code is a simple placement (I have no idea how implement it on Mac)